### DISCUSSION TITLE
How to set tabIndex for Segmented?

### DISCUSSION THREAD

**Discussion by ujbrody** [ant-design/ant-design#39569] [Q&A]
I want to set `tabIndex={-1}` for the `Segmented` component, but it doesn't work. Apparently, the `Segmented` component is using internally an `input` component (`ant-segmented-item-input`), which is still part of the tab flow on the page. How can I reach that input tag and change its tab index to -1?

**Comment by Wxh16144** [comment]
At least for the time being, it doesn't support this, maybe it was designed that way. bummer~
